src/ensembl/src/shared/toggle-image-button/ImageHolder.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { ImageButtonStatus } from 'src/shared/toggle-image-button/ToggleImageButton';
+
+type Props = {
+  buttonStatus: ImageButtonStatus;
+  image: any;
```

Comment:
I think we know what the type of the image is.

If we are passing an svg transformed to React component, that will probably be `ReactNode`, and if we are allowing passing either inline svg or path to image, that will be `ReactNode | string`.

Reply:
Should it be `ComponentClass | string` or `ReactNode | string` ?

I might be doing something wrong but I'm getting the following error when I use `ReactNode`:

> JSX element type 'props.image' does not have any construct or call signatures.

On the line:
`{typeof props.image === 'string' ? <img src={props.image}/> : <props.image/>}`